### PR TITLE
Added cropperjs for cropping profile picture

### PR DESCRIPTION
**Pull request by prakashchoudhary07**
This feature is hidden behind /profile/?dev=true

Cropperjs https://github.com/fengyuanchen/cropperjs/blob/main/README.md

Test for this feature will be sent in the next PR (skipping test for this)

For this feature to work https://github.com/Real-Dev-Squad/website-backend/pull/515/files this should be merged in backend

### CODE REVIEW

File: app/components/image-cropper.hbs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="image-cropper-container">
+  <img
+    class="image-cropper"
+    id="image-cropper"
+    alt="Cropper"
+    src={{this.image}}
+    onload={{this.loadCropper}}
+  />
+</div>
```
Comment (sumitd94) [inline app/components/image-cropper.hbs:9]:
```suggestion
</div>

```

File: app/components/image-cropper.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+import Cropper from 'cropperjs';
+import { action } from '@ember/object';
+
+export default class ImageCropperComponent extends Component {
+  currentImageCropCordinates = null;
```
Comment (harshith-venkatesh) [inline app/components/image-cropper.js:6]: `currentImageCropCordinates` to `currentImageCropCoordinates`?

File: app/index.html [hunk 3]
```diff
@@ -15,6 +15,13 @@
       rel="stylesheet"
       href="{{rootURL}}assets/website-my.css"
     />
+    <link
```
Comment (harshith-venkatesh) [inline app/index.html:18]: We have stylesheet for cropper too , nice!!!

**Review by harshith-venkatesh — APPROVED** [review]

File: app/components/image-cropper.js (hunk 2 above)
Comment (prakashchoudhary07) [reply app/components/image-cropper.js]: Remove this, it was not required, thanks for noticing

File: app/index.html (hunk 3 above)
Comment (prakashchoudhary07) [reply app/index.html:18]: Yes, its good, made it task easy

**Review by sumitd94 — APPROVED** [review]

**Review by swarajpure — CHANGES_REQUESTED** [review]
Can you please add screenshots for mobile and web view both?

**Comment by prakashchoudhary07** [comment]
> Can you please add screenshots for mobile and web view both?
**Mobile view**
Screenshot from 2022-02-02 23-31-25


**Web view**
Screenshot from 2022-02-02 23-36-32

**Review by swarajpure — APPROVED** [review]
LGTM!
Awesome work in here 🔥

**Pull request merged**